packages/contact-center/task/src/Utils/task-util.ts

```diff
@@ -177,20 +176,15 @@ export function getConferenceButtonVisibility(
   webRtcEnabled: boolean,
   isCall: boolean,
   isChat: boolean,
-  isBeingConsulted: boolean
+  isBeingConsulted: boolean,
+  multiPartyConferenceEnabled: boolean
```

Comment:
Should we rather call it conferenceEnabled?

```suggestion
  conferenceEnabled: boolean
```

Reply:
Addressed